CMakeLists.txt

```diff
@@ -202,7 +202,7 @@ elseif(UNIX)
 	else(APPLE)
 		# I wasn't able to do this in any other way.
 		# Things like imported targets result in linking with relative path.
-		set(VGUI_LINK_FLAGS "-L${CMAKE_SOURCE_DIR}/linux/release -l:vgui.so")
```

Comment:
This is `VGUI_LINK_FLAGS`, do you mind moving SDL flags to their own, properly named variable? Also, should it come after or before `-Wl,--no-as-needed`?

Reply:
I set another variable.
As for `-Wl,--no-as-needed` - I'm not sure why it's set in the first place. Probably doesn't matter.